android/app/src/main/java/com/textile/textilenode/TextileNode.java

```diff
@@ -300,4 +305,109 @@ private static void emitDeviceEvent(String eventName, @Nullable WritableMap even
         // https://facebook.github.io/react-native/docs/native-modules-android.html#sending-events-to-javascript
         reactContext.getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class).emit(eventName, eventData);
     }
+
```

Comment:
These are some utility methods to convert to/from `JSONObject` and `WritableMap/Array`. Probably a better place for them?